src/api/payment-requests/payment-requests.md

```diff
@@ -769,7 +769,7 @@ Alternatively you can provide an external transaction Id and the Centrapay [Asse
 {% h4 Example response payload %}
 {% json %}
 {
-  "type": "cancelled",
```

Comment:
`cancelled` is not consistent with other payment activity types such as `payment` or `refund`. Therefore I have updated the type to `cancel`.

Reply:
Yea cancelled is for the paymentRequest and cancel is the activity